bite/BiteManager.cpp

```diff
@@ -45,14 +55,14 @@ void BiteManager::parseBITETransactions(
     for (auto &tx: *_proposal->getTransactionList()->getItems()) {
         try {
             tx->parseAndValidate();
-            auto biteDataField = tx->tryGetBiteData(_currentEpochId);
+            auto biteDataField = tx->tryGetBiteData(_proposal->getEpochID());
```

Comment:
can we cash `BiteData` for every transaction that has it? the same way as we do for `Transaction::parseAndValidate`:
```
auto pt = std::atomic_load(&parsedAndValidatedEthTransaction );
    if (pt)
        return;
```
this way we can avoid using `biteDataFields` map